bin/run_sollve.sh

```diff
@@ -132,16 +147,36 @@ make report_summary  | tail -5 >> abrev.combined-results.txt
 mv results_report results_report45
 
 # Run OpenMP 5.0 Tests
+echo "--------------------------- START OMP 5.0 TESTING ---------------------"
 export MY_SOLLVE_FLAGS="$MY_SOLLVE_FLAGS -fopenmp-version=50"
 make tidy
 make CC=$AOMP/bin/clang CXX=$AOMP/bin/clang++ FC=$AOMP/bin/flang CFLAGS="-lm $MY_SOLLVE_FLAGS" CXXFLAGS="$MY_SOLLVE_FLAGS" FFLAGS="$MY_SOLLVE_FLAGS"  OMP_VERSION=5.0 LOG=1 LOG_ALL=1 VERBOSE_TESTS=1 VERBOSE=1 all
-
-echo "--------------------------- OMP 5.0 Results ---------------------------" >> combined-results.txt
+echo 
+echo "--------------------------- OMP 5.0 Detailed Results ---------------------------" >> combined-results.txt
 echo "--------------------------- OMP 5.0 Results ---------------------------" >> abrev.combined-results.txt
 make report_html
 make report_summary >> combined-results.txt
 make report_summary  | tail -5 >> abrev.combined-results.txt
 mv results_report results_report50
+
+if [ "$ROCMASTER" != "1" ] && [ "$EPSDB" != "1" ]; then
```

Comment:
Just a note here during the EPSDB we have a list of known passing tests so these failing will not be an issue, but we can skip them for now.